seatunnel-connectors-v2/connector-cdc/connector-cdc-mysql/src/main/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/source/MySqlIncrementalSource.java

```diff
@@ -42,9 +42,11 @@
 
 @AutoService(SeaTunnelSource.class)
 public class MySqlIncrementalSource<T> extends IncrementalSource<T, JdbcSourceConfig> {
+    static final String IDENTIFIER = "MySQL-CDC";
```

Comment:
```suggestion
    private static final String IDENTIFIER = "MySQL-CDC";
```

Reply:
<img width="1252" alt="image" src="https://user-images.githubusercontent.com/14371345/209292796-032f37fd-400e-4046-9dfd-135aa66598e0.png">
@TyrantLucifer

Reply:
Got it.